webui/react/src/utils/error.ts

```diff
@@ -135,14 +133,17 @@ const errorLevelMap = {
   [ErrorLevel.Warn]: 'warn',
 };
 
-const openNotification = (e: DetError) => {
-  const key = errorLevelMap[e.level] as keyof NotificationInstance;
-  const notification = antdNotification[key] as (args: ArgsProps) => void;
+const toastSeverityMap = {
+  [ErrorLevel.Error]: 'Error',
+  [ErrorLevel.Fatal]: 'Error',
+  [ErrorLevel.Warn]: 'Warning',
+};
```

Comment:
nit: explicitly type this as `Record<ErrorLevel, Severity>` to prevent needing the cast on line 145